content/minutes/cab/2021-01-19-minutes.md

```diff
@@ -0,0 +1,34 @@
+---
+date: "2021-01-19T00:00:00+00:00"
+title: "Jan 19, 2021 Advisory Board Meeting"
+tags: ["Open Force Field Consortium", "Advisory Board", "minutes"]
+categories: ["Advisory Board Meeting Minutes"]
+draft: false
+description: "Meeting minutes"
+weight: 10
+author: "Diego Nolasco"
+---
+
+The [Open Force Field Consortium Advisory Board](https://openforcefield.org/about/organization/#open-force-field-consortium) met on January 19, 2021.
+The minutes are summarized as follows:
+
+#### Eli Lilly has joined the OpenFF Consortium
+
+* D. Mobley announced that Eli Lilly is now a member of The Open Force Field Consortium. Chuan Tian will act as its representative on the Advisory Board.
+
+#### Updates on Sage release
+
+* D. Mobley announced that the team has been working on applying Wiberg Bond Orders (WBO) into the Sage Project. Generally, they are exploring multiple directions at the same time so that we have a clear path to a more accurate implementation.
+
+#### Industry-led benchmarking study
+
+* G. Tresadern and J. Wagner presented some updates about Geometry Optimization Benchmarking for Industry Partners. D. Dotson and J. Horton are driving the project aiming to <q>obtain performance benchmarks of OpenFF forcefields via geometry optimizations of in-house molecules by industry partners</q>.
```

Comment:
(not blocking) This renders just fine, but I'm curious if there's a reason to do `<q>Quoted text</q>` instead of `"Quoted text"`. I tried the second one locally and it looks the same, but I don't know much about websites so there may be a significant difference.

Reply:
I didn't even try to use " ... " instead of <q> ... </q>. Maybe I'm becoming a HTML enthusiast.